app/models/spree/variant_decorator.rb

```diff
@@ -27,11 +27,6 @@ def join_volume_prices(user = nil)
     end
   end
 
-  # calculates the price based on quantity
-  def volume_price(quantity, user = nil)
```

Comment:
Will this be a major version change on this gem? Has it been released yet?

If so, can we deprecate and still return a value?